docs/HowTo/UpdatePatroni.md

```diff
@@ -0,0 +1,129 @@
+# Patroni Stability Issue Hot Fix
+
+During the upgrade to Openshift 4.6.25 in the Silver cluster during the week of Feb 7, 2022 we observed that app's Patroni clusters that use Patroni `v1.6.51` experienced outages when the cluster nodes where the PostgreSQL pods were running, were restarted as part of the upgrade. The troubleshooting narrowed down the root cause to the [known bug](https://github.com/zalando/patroni/issues/1508) in this Patroni version. It is highly recommended that the teams apply the fix suggested below **as soon as possible** to minimize the risk of service disruption for their database services. 
+
+**The OCP 4.8.25 upgrade will resume on Wed Feb 23, 2022 at 9am**
+
+# Updating Patroni from 1.6.5 to 2.0.1
+
+**Please note: while this fix has been testing multiple times in a lab environment by the Platform Services Team the nature of the bug means that we cannot guarantee that this fix will be completely outage-free. We did repeat the test multiple times and was unable to cause an outage using this method, but this may depend somewhat on your particular deployment. Therefore, please ensure that you test sufficiently in your test environment before applying the upgrade to prod!**
+
+There is a bug in version 1.6.5 of Patroni which can, in some rare circumstances, cause the entire cluser to fall over when one pod goes down, even though other pods may remain up and functioning perfectly. This method for upgrading the patroni version can be used to move past 1.6.5 without triggering this problem during the upgrade.
+
+We recommend moving from 1.6.5 to 2.0.1. If your team wishes to upgrade further, please feel free to do so.
+
+## Step 1: Preparation
+
+Before you begin, as is always best practice, please make sure that you back up your database. 
+
+You may also wish to switch the leader of your Patroni cluster over to pod 0 in your stateful set. This shouldn't be necessary, but will probably make things easier on you.
+
+This list of steps also requires that the stateful set not automatically redeploy on image change. Please ensure that this is correct before proceeding.
+
+Lastly make sure that the current state of your Patroni cluster is healthy, that all members are communicating properly, that you have plenty of space in your PVCs, and that there is a low `Lag in MB` value when you check `patronictl list` to see the status of your cluster. You can run this command from the terminal of any pod in your cluster.
+
+## Step 2: Put your Patroni cluster into Maintenance Mode
```

Comment:
maybe worth to mention what does it mean for application backend to connect to patroni when it's under maintenance mode?

Reply:
done!